versions.mk

```diff
@@ -17,8 +17,8 @@ VERSION ?= v0.6.0
 
 vVERSION := v$(VERSION:v%=%)
 
-CUDA_VERSION := 12.3.2
-GOLANG_VERSION := 1.20.5
+CUDA_VERSION := 12.4.1
```

Comment:
```suggestion
CUDA_VERSION := 12.4.2
```

Reply:
Typo in my commit message . The latest version is 12.4.1